### PR TITLE
test(docs): disable versions check for external contributors

### PR DESCRIPTION
**Pull request by ST-DDT**
Fixes https://github.com/faker-js/faker/pull/2174#issuecomment-1562585163

This disables the versions check for external contributors/those that do not use the faker main repository as their origin.

### CODE REVIEW

**Comment by ST-DDT** [comment]
Result if origin != faker main repo:

grafik

Result if origin = faker main repo:

grafik

File: test/docs/versions.spec.ts [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 import { oldVersions } from '../../docs/.vitepress/versions';
 
-describe('docs versions', () => {
+function isFakerOrigin(): boolean {
+  try {
+    const originUrl = execSync('git remote get-url origin')
+      .toString('utf8')
+      .trim();
+    return (
+      originUrl === 'git@github.com:faker-js/faker' ||
```
Comment (Shinigami92) [inline test/docs/versions.spec.ts:12]: Do you think using `originUrl.includes('faker-js/faker')` is sufficient enough?

**Review by Shinigami92 — COMMENTED** [review]
Will the CI still check the docs version, even when the PR is on another fork?

**Comment by ST-DDT** [comment]
> Will the CI still check the docs version, even when the PR is on another fork?

Good point. I have to check.

File: test/docs/versions.spec.ts (hunk 1 above)
Comment (xDivisionByZerox) [reply test/docs/versions.spec.ts:12]: I don't think it is as you can technically name your fork however you like. So some name that would match your suggestion would simply be `faker-js/faker-fork`. Correct me if I'm wrong tho.

Comment (Shinigami92) [reply test/docs/versions.spec.ts:12]: Yeah, in GitLab you could have that

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
I could be wrong, but it seems that this PR disables the test when a contributor creates a PR in their own fork. But when we then merge that PR, the test would run on our `next` branch and could then fail.

So this doesn't look like it is the right solution, but sadly I also do not have another idea right now :confused:

**Comment by Shinigami92** [comment]
Is the test in our CI always executed via a PR and therefore the origin will always be `faker-js/faker`?
Or does the CI origin run on the fork?

**Comment by ST-DDT** [comment]
I created #2394 as an external contributor test.

- #2394

**Review by Shinigami92 — APPROVED** [review]
Yes, the test gets executed: https://github.com/faker-js/faker/actions/runs/6201950881/job/16839680435?pr=2394#step:8:110

**Comment by ST-DDT** [comment]
From #2394 :

https://github.com/faker-js/faker/actions/runs/6201950881/job/16839679994?pr=2394#step:2:39

> git remote add origin https://github.com/faker-js/faker

https://github.com/faker-js/faker/actions/runs/6201950881/job/16839679994?pr=2394#step:8:108

> stdout | unknown test
> https://github.com/faker-js/faker
> 
> ✓ test/docs/versions.spec.ts  (1 test) 6ms

It uses the remote of the target repository as origin.

**Review by xDivisionByZerox — APPROVED** [review]

**Pull request merged**